debian/build.py

```diff
@@ -40,20 +41,18 @@ def __str__(self):
         self.pkg, self.arch, self.distro, self.version, self.revision)
 
 def main():
-  architectures=('amd64', 'arm', 'arm64')
+  architectures=('amd64',)
```

Comment:
@errordeveloper thought about if we should replace build.py with something else in order to be able to customize those vars for each build